mayastor-test/e2e/node_disconnect/lib/node_disconnect_lib.go

```diff
@@ -105,8 +105,6 @@ func LossWhenIdleTest(nodeToIsolate string, otherNodes []string, disconnectionMe
 	DisconnectNode(nodeToIsolate, otherNodes, disconnectionMethod)
 
 	fmt.Printf("waiting up to 90s for disconnection to affect the nexus\n")
```

Comment:
A nitpick (not a functional problem), but I always try to address magic numbers. It's not a problem per-se, but I like to help readers of the code. I recommend putting 90 to a constant and using it throughout code. https://en.wikipedia.org/wiki/Magic_number_(programming)

Reply:
Agreed, I'll make the change.